src/bots/mascot-bot.js

```diff
@@ -39,6 +46,20 @@ class MascotBot extends SlackBot {
     });
   }
 
+  setTopic(channel, topic, isPublic = true) {
```

Comment:
I didn't name it `channelId` because the parameter for `bot._api` is `channel` and I wanted to just have it as 

```js
{
  channel
}
```

but if it's that important to you I will set it to channelId and do

```js
{
  channel: channelId
}
```